src/libraries/Common/src/System/Net/Http/aspnetcore/Http3/QPack/QPackDecoder.cs

```diff
@@ -413,7 +413,7 @@ private void ProcessHeaderValue(IHttpHeadersHandler handler)
 
             if (_index is int index)
             {
-                Debug.Assert(index >= 0 && index <= H3StaticTable.Instance.Count, $"The index should be a valid static index here. {nameof(QPackDecoder)} should have previously thrown if it read a dynamic index.");
+                Debug.Assert(index >= 0 && index <= H3StaticTable.Count, $"The index should be a valid static index here. {nameof(QPackDecoder)} should have previously thrown if it read a dynamic index.");
```

Comment:
Is the `index <= Count` part is correct?
A valid index is between 0 and Count - 1 (unless there is something special about this case?)